spec/features/flagging_a_content_item_spec.rb

```diff
@@ -4,14 +4,14 @@
   include TaxonomyHelper
   include PublishingApiHelper
 
-  scenario "flagging a content_item as 'I need help'" do
+  scenario "flagging a content_item as 'I need help'", js: true do
```

Comment:
I think we should retain the non-JS scenario if that's what we're meant to be supporting. This is quite a vital part of the application that should still be covered as a feature specs.

Reply:
As we discussed yesterday, the primary tagging functionality only works with select2, so we don't have a non-JS version. This isn't a problem currently as we're providing laptops for the tagathon. Feel free to ignore this comment and the [controller response comment](https://github.com/alphagov/content-tagger/pull/471#discussion_r140239102).